src/Console/CreateInfrastructureCommand.cs

```diff
@@ -19,7 +19,7 @@ public CreateInfrastructureCommand(string name, string description = null) : bas
 
             Add(new Option<string>(new[] { "--site", "-s", }, Resources.INFRASTRUCTURE_SITE_HELP));
             Add(new Option<string>(new[] { "--customer", "-c", }, Resources.INFRASTRUCTURE_CUSTOMER_HELP));
-
+            Add(new Option<bool>(new[] { "--ignoreIfExists", "-ignore", }, Resources.INFRASTRUCTURE_IGNORE_IF_EXISTS_HELP));
```

Comment:
I find it weird to have this flag `-ignore`. All other options that we have, have a long `--...` format and a `-.` short format. I understand that `-i` is usually more associated to _interactive_, not _ignore_, but in case we don't come up with a better letter I would only keep the long format.

Reply:
done :) now only exists one alias ignore-if-exists.